NewPlatform.Flexberry.ServiceBus.RabbitMQ/Components/ReceivingManager/RmqReceivingManager.cs

```diff
@@ -1,26 +1,49 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Configuration;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using ICSSoft.Services;
+using Microsoft.Practices.Unity.Configuration;
 using RabbitMQ.Client;
 using RabbitMQ.Client.Content;
+using Unity;
+using Unity.Resolution;
+
 namespace NewPlatform.Flexberry.ServiceBus.Components
 {
     /// <summary>
     /// Модуль приёма сообщений в формате шины в брокер
     /// </summary>
     internal class RmqReceivingManager : BaseServiceBusComponent, IReceivingManager
     {
-        private IConnectionFactory _connectionFactory;
-
         private IMessageConverter _messageConverter;
 
         private AmqpNamingManager _namingManager;
 
-        public RmqReceivingManager(IConnectionFactory connectionFactory, IMessageConverter converter)
+        public readonly string ConnectionFactoryRegistrationName = "RmqReceivingManagerConnFactory";

```

Comment:
I think it should not be read-only, or it should be specified through the constructor.